standard/docs/en/implementation/serialization.md

```diff
@@ -92,7 +92,9 @@ Whilst this allows complex data to be expressed in flat CSV, users will need to
 
 Instead, data with a one-to-many relationship can be represented using a multi-table serialization. 
 
-Note: OCDS CSV files MUST only be encoded in either UTF-8
+Note
```

Comment:
@duncandewhurst Was the intention to use a `.. note::` admonition?

Reply:
I didn't intend to change the formatting, but yes it makes sense for this note to be in a suitable admonition - I'll add a commit to this branch